docs/adr/0031-rap-status-dokku-service.md

```diff
@@ -0,0 +1,23 @@
+# 31. Rap status update uses Dokku service
+
+Date: 2025-10-23
+
+## Status
+
+Accepted
+
+## Context
+
+The `job-server` needs to store information about the status of every job, so that it can be presented to end-users. Until now, a sync loop in `rap-controller` would regularly post this information to the job-server's jobs API. As part of the RAP API phase 2 work, `job-server` will now retrieve this information from the `rap-controller` using a call to the RAP API. We have created a Django management command `rap_status` which gets this data from the RAP API and stores it in the database.
```

Comment:
nit: un-backtick `job-server` and `rap-controller`? The backticks make it seem (to me at least) like they're referring to code rather than project/system